crates/wash-cli/src/ui/mod.rs

```diff
@@ -68,6 +68,9 @@ async fn ensure_washboard(version: &str, base_dir: PathBuf) -> Result<PathBuf> {
 
 async fn download_washboard(version: &str, install_dir: &PathBuf) -> Result<()> {
     let urls = vec![
+        format!(
+            "https://github.com/wasmCloud/typescript/releases/download/washboard-ui%40{version}/washboard.tar.gz"
+        ),
```

Comment:
When should we remove the old URLs, or should we just keep them indefinitely?

Reply:
Probably on the next release? It doesn't hurt to leave them, aside from a few extra bytes.